configuration/drugs/1020_essential_drugs.csv

```diff
@@ -0,0 +1,835 @@
+cd657ee2-2f5e-4f5e-851d-968c9beb97f4,0,Dapsone 50mg,Dapsone,Comprimé,50mg,,
+e34b5cdc-1435-482d-959a-0db5e92d507c,0,Daunorubicine 20mg/FL,Daunorubicin,Injection,20mg/FL,,
+04963ee4-4604-4074-8fb3-af19f42fb40b,0,Deferrioxamine (chelateur du fer) 100mg/ml,Desferrioxamine,Injection,100mg/ml,,
+5026852b-f285-4732-abad-489b656ef40f,0,Dermobacter mous. pour application locale FL/300ml,Dermobacter,Injection,FL/300ml,,
+2d799342-892a-40ed-933c-c31fcb90c4a8,0,"Dexaméthasone 0,5mg",Dexamethasone,Comprimé,"0,5mg",,
+818add3c-5e27-404b-a63e-5880a002386d,0,Dexaméthasone 4mg/ml,Dexamethasone,Injection,4mg/ml,,
+1a02920b-67e3-4ab2-a797-b929189828e3,0,"Dexaméthasone 5mg, sous blister",Dexamethasone,Comprimé,5mg,,
+17566d31-2173-4953-877b-2a82c093af63,0,Dexamethasone phosphate sodique 4mg/ml 1ml,Dexamethasone Sodium Phosphate,Ampoule,4mg/ml 1ml,,
+39414cf3-0c4c-474d-b6f5-ab4470519afa,0,Dexaméthasone+framycétine (5mg+31 500UI)/5ml,Dexamethasone,Collyre,5mg+31 500UI,,
+7249c3e4-f919-4ccc-b396-8ae7566006eb,0,Dexamethasone+Néomycine+polymyxine B,Dexamethasone,Pommade,,,
+ac58fe11-b9c2-4a67-a1ef-fd2133ce5a7c,0,Dexamethasone+Néomycine+polymyxine B (3mg+10 500UI+18000UI)/3ml,Dexamethasone,Collyre,3mg+10 500UI+18000UI,,
+88587b35-b869-45b5-8c38-11b2c90372bb,0,Dextrométhophane 15mg/ml,Dextrométhophane,Solution,15mg/ml,,
+7f97b227-afef-408a-b4fb-68f458d5be4c,0,Dextrométhophane 4mg,Dextrométhophane,Comprimé,4mg,,
+c5e2457f-fecf-497f-a625-619175da2549,0,Dextrométhophane 5mg/ml,Dextrométhophane,Injection,5mg/ml,,
+14a4ec7c-4028-472a-9f85-42ba2fa3ea93,0,Diazepam 10mg/2ml,Diazepam,Ampoule,10mg/2ml,,
+b6c3a432-3038-4d6d-a704-f8fcfa943dba,0,"Diazepam 5mg, sous blister",Diazepam,Comprimé,5mg,,
+38b56fd6-7789-4627-baf6-1724fedcfba1,0,Diazepam 5mg/ml,Diazepam,Ampoule,5mg/ml,,
+eb34afbb-edd5-48fd-8d36-dd19c5eff331,0,"Diazépam 5mg/ml/0,5ml par voie rectale",Diazepam,Gel,5mg/ml,,
+f9a7a8e8-8599-4af9-a607-6a8e3937c6a3,0,Diazépam 5mg/ml/2ml,Diazepam,Injection,5mg/ml,,
+5e4bd1e8-3a33-4190-96b0-9140202801b8,0,Diclofénac 25mg,Diclofenac,Comprimé,25mg,,
+91a107cc-56c6-4e2b-a3c8-65de210950c9,0,Diclofénac 50mg,Diclofenac,Comprimé,50mg,,
+4027ee86-9e15-4ac2-8e6f-002ffc7c3905,0,Diclofénac 75mg/ml,Diclofenac,Injection,75mg/ml,,
+50dd1890-30e4-4849-b8da-6360752f77a8,0,Diclofénac Sodique 25mg/ml,Diclofenac Sodium,Ampoule,25mg/ml,,
+e7481e2b-9704-4ae0-ade0-aea98334a819,0,Diclofénac Sodique 50mg sous blister,Diclofenac Sodium,Comprimé,50mg,,
+b5370e81-a74c-47ca-9971-d9ac0107f1e7,0,Diclofénac Sodique 75mg/3ml,Diclofenac Sodium,Ampoule,75mg/3ml,,
+65758339-1c9b-4fff-adcc-b6364fcfb563,0,Diethylcabamazine Citrate 50mg sous blister,Diethylcabamazine Citrate,Comprimé,50mg,,
+cbb0d644-e2f7-440f-91c9-b4b3b10e18a2,0,Diéthylcarbamazine 100mg,Diethylcarbamazine,Comprimé,100mg,,
+5ad558ee-f4b7-4310-a212-4f5639f801bd,0,"Diféuines (I, II et III) 23mg+32mg+54mg",Diféuines,Sirop,23mg+32mg+54mg,,
+3fb25c98-6475-4060-b4d3-d791c493bff7,0,"Digoxine 0,25mg",Digoxin,Comprimé,"0,25mg",,
```

Comment:
@lauravignoli small typo on "Digoxine" (missing e). Please see line 285 to 289 as well